src/erc20.sol

```diff
@@ -52,6 +52,7 @@ contract ERC20 {
         wards[msg.sender] = 1;
         symbol = symbol_;
         name = name_;
+        version = 1;
```

Comment:
We should use a constant here. It would be cheaper instead of a storage variable.

Maker did it the same way:
https://github.com/makerdao/dss/blob/master/src/dai.sol#L37